pkgs/development/libraries/libwacom/default.nix

```
@@ -41,7 +41,7 @@ stdenv.mkDerivation rec {
     libgudev
   ];
 
-  doCheck = stdenv.hostPlatform == stdenv.buildPlatform;
+  doCheck = stdenv.hostPlatform == stdenv.buildPlatform && !stdenv.buildPlatform.isRiscV;
```

It would be more robust to do something like this, right?

```suggestion
  doCheck = stdenv.hostPlatform == stdenv.buildPlatform && lib.meta.availableOn stdenv.hostPlatform valgrind;
```